hack/ginkgo-e2e.sh

```diff
@@ -38,12 +38,6 @@ GINKGO_NO_COLOR=${GINKGO_NO_COLOR:-n}
 # If 'y', will rerun failed tests once to give them a second chance.
 GINKGO_TOLERATE_FLAKES=${GINKGO_TOLERATE_FLAKES:-n}
 
-# The number of tests that can run in parallel depends on what tests
-# are running and on the size of the cluster. Too many, and tests will
-# fail due to resource contention. 25 is a reasonable default for a
-# 3-node (n1-standard-1) cluster running all fast, non-disruptive tests.
-GINKGO_PARALLELISM=${GINKGO_PARALLELISM:-25}
```

Comment:
Yes, unused.

Looks like this was replaced with `GINKGO_PARALLEL_NODES` and `GINKGO_PARALLEL`